src/main/java/com/initializer/Employeedetails/ControlApi/Delete.java

```diff
@@ -0,0 +1,70 @@
+package com.initializer.Employeedetails.ControlApi;
+
+import com.initializer.Employeedetails.MessageUtil.MessageUtil;
+import com.initializer.Employeedetails.Repositories.EmployeeRepo;
+import com.initializer.Employeedetails.Repositories.RelationRepo;
+import com.initializer.Employeedetails.Tables.Employee;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.data.domain.Sort;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.stereotype.Service;
+import org.springframework.web.bind.annotation.PathVariable;
+
+import java.util.List;
+
+/******************Service class for deleting records through ID******************/
+
+@Service
```

Comment:
services usually are scoped for the resource and not method. For example, there should be a employee service which does get, delete etc instead of a Get, Delete etc service